src/controller/appController.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+
+import * as userServices from '../services/userServices';
+import * as listServices from '../services/listServices';
+import * as cardServices from '../services/cardServices';
+import * as boardServices from '../services/boardServices';
+
+export const board = async (req: Request, res: Response) => {
```

Comment:
Few things:
- Separate controller in separate files
- Prefer `function` instead of declaring it into varaible.
- While naming function use `verb`
https://www.coding-guidelines.lftechnology.com/docs/javascript/functions